README.md

```diff
@@ -14,7 +14,9 @@ formatting, etc.) can be preserved.
 The syntax of beancount is expected to become slightly less restrictive
 as some missing features are implemented (such as UTF-8 for account names
 and tags on a posting-level).  ledger2beancount aims to be compatible with
-the latest official release of beancount.
+the latest official release of beancount but some functionality may require
```

Comment:
very minor: a comma is missing before "but"

Reply:
Ok